pcsx2/PAD/Linux/SDL/joystick.h

```diff
@@ -49,6 +49,7 @@ class JoystickInfo : public GamePad
 	size_t GetUniqueIdentifier() final;
 
 private:
+	static constexpr uint32_t s_effect_duration_ms = 125; // 125ms feels quite near to original
```

Comment:
That feels like a pretty big hack, is there any reasoning behind this?

Reply:
It's a strange question since it was borrowed from original code:
https://github.com/PCSX2/pcsx2/pull/4027/files#diff-d339082f7e69f44bd7922224b2306f9b8b3a4c2de389f36b36e5e69a8c2e98aeL205

Reply:
oh, my bad, I'm just a dumbass